helper/definitions.py

```diff
@@ -147,9 +147,13 @@ class VideoLinks(str, Enum):
 """Dictionary for the expected time interval of the dialog. For each dialog,
 the first number indicates the minimum expected completion time,
 the second number indicated the maximum expected completion time """

```

Comment:
```suggestion
"""Dictionary for the expected time interval of the dialog. For each dialog,
in case the duration (in minutes) is fixed, it is represented by a single number ,
if the duration is expected to be in a range of values,
the first number indicates the minimum expected completion time,
the second number indicated the maximum expected completion time """

```